pynest/examples/brunel_alpha_nest.py

```diff
@@ -99,7 +100,7 @@ def ComputePSPnorm(tauMem, CMem, tauSyn):
 # Assigning the simulation parameters to variables.
 
 dt = 0.1    # the resolution in ms
-simtime = 1000.0  # Simulation time in ms
+simtime = 100.0  # Simulation time in ms
```

Comment:
This change seems unrelated. Please remove (or provide an explanation why it should be here).

Reply:
yes, this was added in a mistake.